examples/pwm/pwm_main.c

```diff
@@ -307,6 +307,13 @@ static void parse_args(FAR struct pwm_state_s *pwm, int argc,
   for (index = 1; index < argc; )
     {
       ptr = argv[index];
+      if (ptr[0] == '>')
```

Comment:
should '>' be processed by nsh? Why we need special handle here?

Reply:
Maybe `CONFIG_FILE_STREAM` is not set, so `>` is getting passed to built-in app?

Reply:
Unfortunately CONFIG_FILE_STREAM is enabled and the issue is happening.

Reply:
Could you please post `defconfig` file to the issue?

Reply:
Hi Petro, I discovered that the issue was happening because I was using an old NuttX version:
NuttX:
19beb307dd5023c2bf414dc54c3141bfc95c7251

Apps:
c222043ed1ede9ef8e9e93ffed14ee9c41c4a2b1

This is may .config: https://pastebin.com/raw/duUsX6hX